hazelcast/src/main/java/com/hazelcast/jet/impl/JobCoordinationService.java

```diff
@@ -1201,7 +1205,8 @@ private JobExecutionRecord ensureExecutionRecord(long jobId, JobExecutionRecord
         return record != null ? record : new JobExecutionRecord(jobId, getQuorumSize());
     }
 
-    @SuppressWarnings("WeakerAccess") // used by jet-enterprise
+    @SuppressWarnings("WeakerAccess")
+        // used by jet-enterprise
```

Comment:
Did you use autoformat? This looks weird.

Reply:
Yes. Reverted all formatting changes.